src/components/BeerDetails.js

```diff
@@ -0,0 +1,75 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { Button, Card, CardImg, Col, Container, Row } from "react-bootstrap";
+import { Link, useParams } from "react-router-dom";
+
+function BeerDetails() {
+  const [beer, setBeer] = useState([]);
+  const { id } = useParams();
+  const baseURL = "https://ih-beers-api2.herokuapp.com/beers";
+
+  useEffect(() => {
+    axios
+      .get(`${baseURL}/${id}`)
+      .then((response) => {
+        setBeer(response.data);
+      })
+      .catch((e) => console.log(e.response.data));
+  });
```

Comment:
With that addition the random stops looping through

```suggestion
  }, []);
```